crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -186,6 +192,11 @@ impl SweetConductor {
         Self::from_config(standard_config()).await
     }
 
+    /// Get the rendezvous config that this conductor is using, if any
+    pub fn get_rendezvous_config(&self) -> Option<DynSweetRendezvous> {
+        self._rendezvous.clone()
```

Comment:
If we're using this now, let's remove the underscore.

```suggestion
        self.rendezvous.clone()
```